tests/ZFToolTest/Diagnostics/TestAsset/AlwaysFailCheck.php

```diff
@@ -0,0 +1,16 @@
+<?php
+namespace ZFToolTest\Diagnostics\TestAsset;
+
+use ZendDiagnostics\Check\AbstractCheck;
+use ZendDiagnostics\Check\CheckInterface;
+use ZendDiagnostics\Result\Failure;
+
+class AlwaysSuccessCheck extends AbstractCheck implements CheckInterface
```

Comment:
Please add just a docblock explaining what this asset is supposed to do and how it is supposed to be used